sympy/vector/deloperator.py

```diff
@@ -13,20 +9,11 @@ class Del(Basic):
     mathematical expressions as the 'nabla' symbol.
     """
 
-    def __new__(cls, system):
-        if not isinstance(system, CoordSysCartesian):
-            raise TypeError("system should be a CoordSysCartesian")
-        obj = super(Del, cls).__new__(cls, system)
-        obj._x, obj._y, obj._z = system.x, system.y, system.z
-        obj._i, obj._j, obj._k = system.i, system.j, system.k
-        obj._system = system
-        obj._name = system.__str__() + ".delop"
+    def __new__(cls, ):
+        obj = super(Del, cls).__new__(cls)
+        obj._name = "delop"
```

Comment:
`Del` was only created inside `CoordsysCartesian` so I think that warning only inside `CoordSysCartesian` should be enough